alarm/utils.py

```diff
@@ -0,0 +1,112 @@
+import json
+import ssl
+
+import websocket
+from sentry_sdk import capture_exception
+from queue import Queue
+from alarm.models import Coin
+
+previous_close_price = None
+previous_candlestick = None
+current_prices = None
+previous_prices = None
+
+
+def get_binance_price_and_analized(currencies, intervals):
+    websocket.enableTrace(False)
+    sockets = [f'wss://stream.binance.com:9443/ws/{currency}@kline_{interval}' for currency, interval in
```

Comment:
How we check that user provided coin name is valid and exists?

Reply:
Can we get info about all coins with one socket?

Reply:
1. It's will be do it systemd

2. Yes, I think

Reply:
UPD: Add in TODO: validation of name of coin in model